### PR TITLE
SlimBanner: implement below small breakpoint start alignment of elements

### PR DESCRIPTION
**Pull request by AlbertCarreras**
#### What changed?

SlimBanner: implement below small breakpoint start alignment of elements

NOTE: The implementation should be refactor taking advantage of Flex responsive props.

#### BEFORE

Screen Shot 2022-11-22 at 3 12 41 PM

#### AFTER

screen_shot_2022-11-22_at_3 03 27_pm

### Links

- [Jira](https://jira.pinadmin.com/browse/GESTALT-5068)

### CODE REVIEW

File: packages/gestalt/src/SlimBanner.js [hunk 1]
```diff
@@ -176,68 +176,95 @@ export default function SlimBanner({
...
-              size={16}
-            />
-          </Flex.Item>
-        )}
+  const startIcon =
+    !isDefault && iconAccessibilityLabel ? (
+      <Flex.Item alignSelf={shouldShowButtons ? undefined : 'start'}>
+        <Icon accessibilityLabel={iconAccessibilityLabel} color={iconColor} icon={icon} size={16} />
+      </Flex.Item>
+    ) : null;
```
Comment (AlbertCarreras) [inline packages/gestalt/src/SlimBanner.js:184]: this null is needed, it was just added. prevent ghost spacing

**Review by ponori — APPROVED** [review]
This is looking really good from a design perspective Alberto  - nice work!

I think there's still some room to noodle on the spacing/padding, but I'd argue that's outside the scope of this work.

File: packages/gestalt/src/SlimBanner.js [hunk 2]
```diff
@@ -176,68 +176,97 @@ export default function SlimBanner({
...
+  ) : null;
+
+  const actions = !isBare && primaryAction && (
+    <Box display="flex" mdDisplay="none" flex="none" alignSelf="end" marginTop={4}>
+      <PrimaryAction {...primaryAction} />
     </Box>
   );
+
+  return (
+    <Fragment>
```
Comment (ayeshakmaz) [inline packages/gestalt/src/SlimBanner.js:228]: This method is going to create issues for testing by duplicating all the elements in the DOM. Can we do this instead with CSS? I'll reach out tomorrow, but let's not merge this yet

**Review by ayeshakmaz — CHANGES_REQUESTED** [review]

File: packages/gestalt/src/SlimBanner.js (hunk 2 above)
Comment (AlbertCarreras) [reply packages/gestalt/src/SlimBanner.js:228]: Ok... le me exploreee

**Pull request closed without merging**